app/assets/javascripts/openseadragon/rails.js

```diff
@@ -15,5 +15,11 @@
       handler = 'page:load ready';
```

Comment:
Does this branch also need some handling?

Reply:
Yes, thanks. I saw "Turbolinks", and my eyes moved right over it...